lib/server.coffee

```diff
@@ -166,7 +166,7 @@ module.exports = exports = (argv) ->
   # including hbs to use handlebars/mustache templates
   # saved with a .html extension, and no layout.
 
-  app.set('views', path.join(__dirname, '..', '..', 'wiki-client', '/views'))
+  app.set('views', path.join(require.resolve('wiki-client'), '..', 'views'))
```

Comment:
I think this is a good path forward! And it reminded me of a good trick for dealing with package relative paths -  instead of resolving `wiki-client` resolve `wiki-client/package.json` You know it always has to be there (or it can't get published to npm) and it's always in the same path location (unlike `wiki-client` which could be changed by setting `main` in the package.json), as a bonus it should make the corresponding `wiki-client` change unnecessary :)

Reply:
Thanks @nrn for the excellent trick. I like this a lot better not needing the change in wiki-client!